docs/CUSTOMIZATION.md

```diff
@@ -0,0 +1,140 @@
+There is also `setSelectedDate()` will will clear the current selection(s) and select the provided date.
+
+There are also: `clearSelection()`, `getSelectedDates()`, and `getSelectionMode()`; which should be self-explanitory.
+
+
+## Showing Other Dates
+
+By default, only days of one month, in the min-max range, are shown.
+You can customize this by setting `mcv_showOtherDates` in xml, or by calling `setShowOtherDates()`.
+
+In xml, there are flags for `other_months`, `out_of_range`, and `decorated_disabled`, and also `none`, `all`, or `defaults` for convenience.
+
+There are similar constants on `MaterialCalendarView` such as `MaterialCalendarView.SHOW_DECORATED_DISABLED`.
+
+
+## Selection Color
+
+The default color of the calendar selector is the one set referenced by `?android:attr/colorAccent` on 5.0+ or `?attr/colorAccent` from the AppCompat library (black as a last resort).
+You can provide a custom color by setting `mcv_selectionColor` in xml, or by calling `setSelectionColor()`.
+
+If you want more control than just color, you can use the [decorator api](DECORATORS.md) to set a [custom selector](CUSTOM_SELECTORS.md).
+
+
+## First Day Of The Week
+
+The default first day of the week is Sunday. You can set a custom day of the week by setting `mcv_firstDayOfWeek` in xml, or by calling `setFirstDayOfWeek()`.
+The xml attribute is an enum of `sunday` through `saturday` and `setFirstDayOfWeek()` accepts values from `java.util.Calendar` such as `Calendar.MONDAY`.
+
+
+## Date Ranges
+
+By default, the calendar displays months for 200 years before today and 200 years after.
+You can specify different minimum and maximum dates by calling `setMinimumDate(CalendarDay)` and `setMaximumDate(CalendarDay)`.
+Passing `null` will reset back to the default 200 years.
+There are also convience methods that accept a `Calendar` or a `Date` object, which convert them to a `CalendarDay` using the relevant `CalendarDay.from()` factory method.
+
+
+## Topbar Options
+
+### Visibility
+
+You can hide or reshow the topbar (arrow buttons and month label) by calling `setTopbarVisible(boolean)`
+
+### Arrow Color
+
+You can change the topbar arrow colors by setting `mcv_arrowColor` in xml, or calling `setArrowColor(int)`.
+This will color the left and right arrow masks using a color filter.
+
+### Arrow Masks
+
+Arrow masks are drawables that will be colored with arrow color and be drawn for previous and next month buttons.
+The color is applied with a color filter using `PorterDuff.Mode.SRC_ATOP`.
+They can be set in xml using `mcv_leftArrowMask` and `mcv_rightArrowMask` or by calling `setLeftArrowMask()` and `setRightArrowMask()`.
+
+
+## Custom Labels
+
+### Header
+
+You can customize the label displayed in the header by setting a custom `TitleFormatter` by calling `setTitleFormatter()`.
+You will be passed a `CalendarDay` with a month and year set, and you can format it to any `CharSequence`.
```

Comment:
This is confusing. Does the setTitleFormatter method return this? How will it be passed to us?